server/webapp/WEB-INF/rails.new/app/assets/stylesheets/css/build_detail.scss

```diff
@@ -290,22 +290,63 @@ div.build_detail_container_content {
 .widget iframe {
     border: none;
 }
+
+#console-action-bar {
+    min-height: 20px;
+    padding: 10px;
+
+    background-color: #333;
+
+    div {
+        float: right;
+    }
+
+    a {
+        padding-right: 15px;
+        color: yellow;
+    }
+
+    &.white-theme {
+        background-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.